assets/wizards/setup/views/welcome/index.js

```diff
@@ -258,27 +311,68 @@ const Welcome = ( { buttonAction } ) => {
 
 				{ errors.length ? errors.map( renderErrorBox ) : null }
 				{ ( isInit || isDone ) && (
-					<Card noBorder className="newspack-card__footer">
-						{ isInit ? (
-							<CheckboxControl
-								checked={ shouldInstallStarterContent }
-								label={ __( 'Install demo content', 'newspack' ) }
-								onChange={ setShouldInstallStarterContent }
-							/>
-						) : (
-							<div />
+					<>
+						<GlobalNotices />
+						{ isInit && (
+							<>
+								<ButtonCard
+									href="#"
+									title={ __( 'Start a new site', 'newspack' ) }
+									desc={ __( "You don't have content to import", 'newspack' ) }
+									icon={ addCard }
+									className="br--top"
+									isPressed={ buttonNew }
+									onClick={ () => setButtonNew( true ) }
+									grouped
+								/>
+								<ButtonCard
+									href="#"
+									title={ __( 'Migrate an existing site', 'newspack' ) }
```

Comment:
This importer is handling WP only:

```suggestion
									title={ __( 'Migrate an existing WordPress site', 'newspack' ) }
```

Reply:
Fixed: https://github.com/Automattic/newspack-plugin/pull/1077/commits/c6fb603523153bebf851b0d57c990595baff6435